.github/workflows/build.yml

```diff
@@ -165,6 +165,7 @@ jobs:
       - integration-test
       - image-scan
     runs-on: ubuntu-22.04
+    if: ${{ ! startsWith(github.ref, 'dependabot/')  && ! startsWith(github.head_ref, 'dependabot/')}}
```

Comment:
Do we need both checks here?

Reply:
in a push, the github.ref is the fully formated branch name like `refs/heads/<branch_name>` (so i just realized that i have to add the "refs/heads/" part in order for it to work when something is pushed.
In a pull request, the github.ref is the merge branch like `refs/pull/<pr_number>/merge ` so this check doesn't work in pull requets.

head_ref is the source branch name of the pull request, but it only works in pull requests.

See: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

So if we want to have the check in both pushes and pull requests, both checks are needed. But if we only want to check in pull requests, the right part would be enough.

Reply:
As the dependant is doing only PR's the second check will be sufficient.